### PR TITLE
Implementation of Logs Tail Lines

### PR DESCRIPTION
**Pull request by daniel-de-vera**
Implementation of:

```bash
signadot logs --job <job-name> --tail <uint>
```

And some other minor improvements.

### CODE REVIEW

File: internal/config/logs.go [hunk 1]
```diff
@@ -7,13 +7,15 @@ import (
...
+	TailLines uint
 }
 
 func (c *Logs) AddFlags(cmd *cobra.Command) {
-	cmd.Flags().StringVarP(&c.Job, "job", "j", "", "job name where to get the attempt Logs")
+	cmd.Flags().StringVarP(&c.Job, "job", "j", "", "job name whose log lines will be displayed")
 	cmd.MarkFlagRequired("job")
 
-	cmd.Flags().StringVarP(&c.Stream, "stream", "s", "stdout", "channel where get the logs stdout or stderr")
+	cmd.Flags().StringVarP(&c.Stream, "stream", "s", "stdout", "standard stream from where to display log lines (stdout or stderr)")
```
Comment (scott-cotton) [inline internal/config/logs.go:19]: would suggest to remove the word "standard" from "standard stream"

File: internal/command/logs/command.go [hunk 2]
```diff
@@ -54,34 +62,36 @@ func list(cfg *config.Logs, out io.Writer) error {
...
-				fmt.Print(log.Message)
 			case "error":
 				return errors.New(string(event.Data))
-			case "signal":
 
+			case "signal":
 				switch string(event.Data) {
 				case "EOF":
-					fmt.Println()
+					out.Write([]byte("\n"))
```
[the remark this answers is not in the document]
Comment (daniel-de-vera) [reply internal/command/logs/command.go]: It's done here: https://github.com/signadot/cli/pull/122/commits/9cb8a136216ca571c3906643f2272c647e7aada4.

File: internal/config/logs.go (hunk 1 above)
Comment (daniel-de-vera) [reply internal/config/logs.go]: It's done here: https://github.com/signadot/cli/pull/122/commits/9cb8a136216ca571c3906643f2272c647e7aada4.

**Review by scott-cotton — APPROVED** [review]
LGTM

**Pull request merged**